<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QWidget" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:vertical {
    background: red;
    position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
    left: 4px; right: 4px;
}

QSlider::handle:vertical {
    height: 10px;
    background: green;
    margin: 0 -4px; /* expand outside the groove */
}

QSlider::add-page:vertical {
    background: white;
}

QSlider::sub-page:vertical {
    background: pink;
}
/*  QLabel */
QFrame, QLabel, QToolTip {
    border: 2px solid green;
    border-radius: 4px;
    padding: 2px;
   text-align: center;
}
/*  Button */
QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
QPushButton:open { /* when the button has its menu open */
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton::menu-indicator {
    image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom right;
}

QPushButton::menu-indicator:pressed, QPushButton::menu-indicator:open {
    position: relative;
    top: 2px; left: 2px; /* shift the arrow by 2 px */
}

QTextEdit, QListView {
    background-color: white;
    background-image: url(draft.png);
    background-attachment: scroll;
}

/* LCD*/
QLCDNumber {
     color: red;
     background-color: rgb(0, 170, 255) 
}
/* The result */
QLabel[IsResult = true]{
     color: red;
     background-color: rgb(220, 220, 220) 
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Group of Neural Net</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_showNet">
          <property name="text">
           <string>Show Net</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_saveNet">
          <property name="text">
           <string>Save Net</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_createNet">
          <property name="text">
           <string>Create Network</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_reTrain">
          <property name="text">
           <string>Re-Train</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_loadModelNet">
          <property name="text">
           <string>Load Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_predict">
          <property name="text">
           <string>Predict</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_NeuralNet">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btn_show">
         <property name="text">
          <string>Show Feature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>Save Feature</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_lookuptable">
         <property name="text">
          <string>Config LookUpTable 1D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_showFeatureIndex">
         <property name="text">
          <string>Show Feature Index 1D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="featureRange">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>1D Feature</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>2D Feature</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_lookuptable2D">
          <property name="text">
           <string>Config LookUpTable 2D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_showFeatureIndex2D">
          <property name="text">
           <string>Show Feature Index 2D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="featureRange2D">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="text2Dfeature">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
